### ISSUE TITLE
[SF][A11y][Switch] accessibleName/accessibleNameRef does not work

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#4887]
### **Bug Description**
I expect to use `accessibleName`/`accessibleNameRef` to specify customized accessible name for switch which could be announced by screen reader. It applied to `aria-label` attribute of the final dom element, however, it does not be announced by screen reader becaused `aria-labelledby` attribute is also applied (see the [code](https://github.com/SAP/ui5-webcomponents/blob/master/packages/main/src/Switch.hbs#L7)). However, `aria-labbeledby` identify the [hidden text](https://github.com/SAP/ui5-webcomponents/blob/master/packages/main/src/Switch.hbs#L50) which is not expected to be announced.

### **Expected Behavior**
`accessibleName`/`accessibleNameRef` specified to the ui5-switch should be announced by screen reader.

Take an example below, I expect the screen reader can announce "Status Active" (not "On").

<img width="1143" alt="Screen Shot 2022-03-11 at 10 11 37 PM" src="https://user-images.githubusercontent.com/1751846/157886377-600cb2ea-ae69-4bb7-bf8e-8d5dadf38633.png">

### **Context**
- UI5 Web Components version: 1.2.1
- OS/Platform: macOS
- Browser: Chrome
- Affected component: ui5-switch

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High
